kalite/settings/base.py

```diff
@@ -461,9 +461,10 @@
 # Separate session caching from file caching.
 SESSION_ENGINE = getattr(
     local_settings, "SESSION_ENGINE", 'django.contrib.sessions.backends.signed_cookies' + (''))
-    
-# Expire session cookies whenever we close the browser.
-SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+
+# Expire session cookies after 30 minutes, but extend sessions when there's activity from the user.
+SESSION_COOKIE_AGE = 60 * 30     # 30 minutes
+SESSION_SAVE_EVERY_REQUEST = True
```

Comment:
I have not measured, unfortunately. And the system @foolchan2556 made isn't ready yet. But hopefully we can get one by the end of 0.16.x. 

By my cursory testing though there's no noticeable performance change.